src/components/chips/chips.scss

```diff
@@ -0,0 +1,28 @@
+md-chips {
+  md-chips-wrap {
+    li {
```

Comment:
Try to avoid using element selectors for styles and just add a class. This makes it easier for people to override the style and makes it easier to create a CSS API.